.github/filters.yml

```
@@ -4,3 +4,5 @@ ingestion_server:
   - ingestion_server/**
 frontend:
   - frontend/**
+  - package.json
+  - pnpm-lock.yaml
```

Should these files also be included?
```suggestion
frontend:
  - frontend/**
  - package.json
  - pnpm-lock.yaml
  - pnpm-workspace.yaml
  - .npmrc
  - .pnpmfile.cjs
```